libres/lib/enkf/enkf_state.cpp

```diff
@@ -464,120 +466,115 @@ static forward_load_context_type * enkf_state_alloc_load_context(const ensemble_
    Will mainly be called at the end of the forward model, but can also
    be called manually from external scope.
 */
-static int enkf_state_internalize_results(ensemble_config_type * ens_config,
-                                          model_config_type * model_config,
-                                          const ecl_config_type * ecl_config,
-                                          const run_arg_type * run_arg,
-                                          stringlist_type * msg_list) {
-
-  forward_load_context_type * load_context = enkf_state_alloc_load_context( ens_config, ecl_config, run_arg, msg_list);
-  /*
+static int enkf_state_internalize_results(ensemble_config_type *ens_config,
+                                          model_config_type *model_config,
+                                          const ecl_config_type *ecl_config,
+                                          const run_arg_type *run_arg,
+                                          stringlist_type *msg_list) {
+
+    forward_load_context_type *load_context = enkf_state_alloc_load_context(
+        ens_config, ecl_config, run_arg, msg_list);
+    /*
```

Comment:
Indentation of comment now looks weird because the indentation of the code has changed.